web/src/HUD.tsx

```diff
@@ -207,16 +202,16 @@ class HUD extends Component<HudProps, HudState> {
   render() {
     let view = this.state.view
 
-    let needsNudge = view ? view.needsAnalyticsNudge : false
-    let resources = (view && view.resources) || []
+    let needsNudge = view?.needsAnalyticsNudge
+    let resources = view?.resources
```

Comment:
are you sure this is right? i think you still need the `|| []` if resources is null

Reply:
I agree that this isn't a 1 to 1 translation of this behavior, however according to the type definitions the resources field can't be null, and it worked fine in my testing locally.

Reply:
Oh wait, I misunderstood this feature. if `view` is `undefined|null` `view?.` returns undefined. So yes we still need to handle that. Thanks for second guessing me!